config/action.d/mail-whois-lines.conf

```diff
@@ -42,7 +42,7 @@ actionban = printf %%b "Hi,\n
             Here is more information about <ip>:\n
             `whois <ip> || echo missing whois program`\n\n
             Lines containing IP:<ip> in <logpath>\n
-            `grep -E '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n
+            `grep -E <grepopts> '(^|[^0-9])<ip>([^0-9]|$)' <logpath>`\n\n
```

Comment:
My bad!  I thought that -E serves as -e, i.e. requires the actual extended regexp as its argument.  But it doesn't -- just a flag....  so it is fine as is.  I would probably keep -E still within the actual definition since it is (is it? ;)   ) mandatory for coded there regexp.